context/src/junit/bindings.rs

```diff
@@ -11,13 +11,16 @@ use quick_junit::{
 #[cfg(feature = "wasm")]
 use wasm_bindgen::prelude::*;
 
+const MICROSECONDS_PER_SECOND: i64 = 1_000_000;
+
 #[cfg_attr(feature = "pyo3", gen_stub_pyclass, pyclass(get_all))]
 #[cfg_attr(feature = "wasm", wasm_bindgen(getter_with_clone))]
 #[derive(Clone, Debug)]
 pub struct BindingsReport {
     pub name: String,
     pub uuid: Option<String>,
     pub timestamp: Option<i64>,
+    pub timestamp_micros: Option<i64>,
```

Comment:
Do we really need micro second precision?

Reply:
Really need, not sure. But it's what our python date parser was giving us and was just going for parity to start